spec/index.md

```diff
@@ -335,163 +325,131 @@ Each dimension is represented by a key in the `dimensions` object. The object ha
 }
 ```
 
-## Examples
```

Comment:
My bad - i looked through but missed this 😄 

All looks good and agree with moving.